articles/azure-functions/functions-scenario-database-table-cleanup.md

```diff
@@ -56,7 +58,7 @@ A function app hosts the execution of your functions in Azure. It is a best prac
     | Setting       | Suggested value | Description             | 
     | ------------ | ------------------ | --------------------- | 
     | **Name**  |  sqldb_connection  | Used to access the stored connection string in your function code.    |
-    | **Value** | Copied string  | Past the connection string you copied in the previous section. |
+    | **Value** | Copied string  | Paste the connection string you copied in the previous section and replace `{your_username}` and `{your_password}` placeholders with real values. |
```

Comment:
Typo with "Past"
Also, the copied connection string has placeholder values, so indicate that user has to replace with real values

Reply:
@andresnava good catch! Thanks. 😁